src/main/java/uk/gov/hmcts/reform/civil/utils/CaseQueriesUtil.java

```diff
@@ -5,37 +5,39 @@
 import uk.gov.hmcts.reform.civil.model.querymanagement.CaseQueriesCollection;
 import uk.gov.hmcts.reform.civil.model.querymanagement.LatestQuery;
 
-import java.util.Comparator;
 import java.util.List;
+import java.util.Optional;
 
-import static uk.gov.hmcts.reform.civil.utils.ElementUtils.unwrapElements;
+import static uk.gov.hmcts.reform.civil.utils.UserRoleUtils.isApplicantSolicitor;
+import static uk.gov.hmcts.reform.civil.utils.UserRoleUtils.isRespondentSolicitorOne;
+import static uk.gov.hmcts.reform.civil.utils.UserRoleUtils.isRespondentSolicitorTwo;
 
 public class CaseQueriesUtil {
 
+    private static final String UNSUPPORTED_ROLE_ERROR = "User does have a supported case role for query management.";
```

Comment:
@Gareth40343 does this message should contain text like "....unsupported case role ..."?